flutter_local_notifications/example/lib/main.dart

```diff
@@ -926,22 +926,24 @@ class _HomePageState extends State<HomePage> {
           ),
           TextButton(
             onPressed: () async {
-              await flutterLocalNotificationsPlugin.zonedSchedule(
-                  0,
-                  'scheduled title',
-                  'scheduled body',
-                  tz.TZDateTime.now(tz.local).add(const Duration(seconds: 5)),
-                  const NotificationDetails(
-                      android: AndroidNotificationDetails(
-                          'full screen channel id', 'full screen channel name',
-                          channelDescription: 'full screen channel description',
-                          priority: Priority.high,
-                          importance: Importance.high,
-                          fullScreenIntent: true)),
-                  androidAllowWhileIdle: true,
+              await flutterLocalNotificationsPlugin.scheduleNotification(
```

Comment:
AFAIK the scheduleNotification methods are deprecated, zonedSchedule should be used